.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,58 @@
+<!--
+    Hello! Thanks for submitting a pull request to MusicBrainz Server
+    We appreciate your time and interest in helping our project!
+
+    Use this template to help us review your change. Not everything is required,
+    depending on your change. Keep or delete what is relevant for your change.
+    Remember that it helps us review if you give more helpful info for us to
+    understand your change.
+
+    Ensure that you’ve read through and followed the Contributing Guidelines, in
+    https://github.com/metabrainz/musicbrainz-server/blob/master/CONTRIBUTING.md
+-->
+
+# Summary
+<!--
+    Just give us a overview of your change, don’t go into technical details yet.
+-->
+
+:beginner: Describe this change in 1-2 sentences
+
+
+# Problem
+<!--
+    Anything that helps us understand why you are making this change goes here.
+    What problem are you trying to fix? What does this change address?
+-->
+
+:beginner: Reference related tickets with MBS-XXX at least.
```

Comment:
"At the very least add a link to the related ticket." ? (MBS-XXX doesn't get turned into a link, so it's annoying - an actual link is much better).

Reply:
I prefer MBS-XXXXX since an actual reference might not be correctly replaced by beginners (or just be forgotten even by us). As an author or a reviewer, it is easier to distinguish this example from a real reference.

Reply:
> MBS-XXX doesn't get turned into a link

Maybe I overthought your comment, maybe just you forgot about MBS/STYLE references being automatically turned into actual links since October:
- https://chatlogs.metabrainz.org/brainzbot/metabrainz/msg/4468833/
- https://chatlogs.metabrainz.org/brainzbot/metabrainz/msg/4472256/

Reply:
My point is just "MBS-123" doesn't actually become a link in the PR text. It does on the commit title and whatnot, but that means you still need to go searching for the actual link rather than clicking directly from the PR description.

Reply:
It does since October, bitmap and me use it at least.

Reply:
Ok then!